src/api/Gemfile.lock

```diff
@@ -273,6 +273,9 @@ GEM
       childprocess (~> 5.0)
       logger (~> 1.6)
     lint_roller (1.1.0)
+    liquid (5.11.0)
```

Comment:
@hellcp-work the gem file in "vendor/cache" and the version set in the Gemfile don't match, I guess maybe happened during rebasing. also the version of the strscan gem shouldn't be updated since its part of the ruby std gems (we need to keep the same version that is used by the ruby interpreter in our bundle)